style/style.css

```
@@ -0,0 +1,132 @@
+p,
+a,
+button,
+h3,
+h2,
+h1 {
+    font-family: Roboto;
+    font-weight: 400;
+    font-size: 16px;
+    color: #727272;
+    margin: 0;
+}
+
+h1,
+h2 {
+    color: #000;
+    font-size: 48px;
+    font-weight: 500;
+}
+
+h3 {
+    font-size: 21px;
+    margin-bottom: 5px;
+}
+
+li {
+    list-style-type: none;
+    display: inline-block;
+    margin-right: 42px;
+}
+
+a {
+    text-decoration: none;
+    color: #000;
+}
+
+img {
+    width: 50px;
+    float: left;
+    margin-right: 20px;
+}
+
+footer {
+    padding-top: 50px;
+    padding-bottom: 50px;
+    color: #FFF;
+}
+
+.wrap {
+    width: 1120px;
+    margin: 0 auto;
+}
+
+.first-block, .block-four {

```

```suggestion
.first-block,
.block-four {

```

так більщ readable як на мене )